tests/test.purchase.jsx

```diff
@@ -97,6 +110,13 @@ describe('Purchase', function() {
       }),
     });
 
+    store.dispatch({
+      type: actionTypes.RX_BRAINTREE_TOKEN,
+      user: Object.assign({}, defaultUser, {
+        braintreeToken: 'braintree-token',
+      }),
+    });
+
     store.dispatch(purchaseActions.payWithNewCard());
 
```

Comment:
if you add a quick test here that makes sure the BraintreeToken component is rendered when `user.braintreeToken` is null then we'll have coverage of an important part of the payflow logic.